omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java

```diff
@@ -17,13 +22,25 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static org.openmrs.module.spa.filter.SpaFilter.DEFAULT_SPA_BASE_URL;
```

Comment:
I would put these in a constants class SpaConstants like this one: https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/util/OpenmrsConstants.java

Reply:
@dkayiwa I have introduced the constants class

Reply:
@dkayiwa I have introduced the constants class